extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -126,6 +125,10 @@ export class AmpUserNotification extends AMP.BaseElement {
       assertHttpsUrl(this.dismissHref_, this.element);
     }
 
+    /** @private @const {boolean} */
+    this.persistentDismissal_ = this.element.getAttribute(
+            'persistent-dismissal') != 'false';
```

Comment:
"false" are not very common in html, are they? Seems "yes" and "no" are more common. Maybe support both?